third_party/move/move-model/src/metadata.rs

```diff
@@ -19,6 +19,10 @@ use std::{
 const UNSTABLE_MARKER: &str = "-unstable";
 
 pub static COMPILATION_METADATA_KEY: &[u8] = "compilation_metadata".as_bytes();
+pub static DEFAULT_COMPILER_VERSION_FOR_V1: CompilerVersion = CompilerVersion::V1;
```

Comment:
Could we do it like in this PR? This is nicer I think. The PR only does it for language, but we can do it for compiler as well.

https://app.graphite.dev/github/pr/aptos-labs/aptos-core/14964/move-decompiler-Model-import-from-binary-and-basic-decompiler-setup#file-third_party/move/move-model/src/metadata.rs

Reply:
done